source/ceylon/buffer/codec/IncrementalCodec.ceylon

```diff
@@ -23,8 +22,10 @@ shared class ChunkConvert<ToMutable, FromImmutable, ToSingle, FromSingle>(conver
     shared Boolean done => remainder.empty;
```

Comment:
The converter is satisfied then. The user needs to know if stuff is in remainder, and exposing remainder directly would be bad for encapsulation.